assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -71,7 +71,7 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi, encodeMarkup }) => {
       }
 
       const query = {
-        _embed: 'author',
+        _embed: 'wp:lock,wp:lockuser,author',
```

Comment:
Is there a particular reason why these two are prefixed with `wp:`, whereas `author` isn't?

Reply:
Just copying WP core naming here.

Reply:
Where does WP core use that naming? And why doesn't it use that for `author`?

`lock` and `lockuser` is something that we build, so having it prefixed with `wp:` is confusing.

Reply:
WordPress does use this wp: prefix. See https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/rest-api/endpoints/class-wp-rest-block-directory-controller.php#L153-L179

I thought there were defined list of allowed link types, like author. Everything else had to be prefixed with wp: 

https://jsonapi.org/format/#document-links

Honestly never really looked into why this is the case.

Reply:
Ah, indeed. https://developer.wordpress.org/rest-api/using-the-rest-api/linking-and-embedding/ explains it, pointing to allowed list at http://www.iana.org/assignments/link-relations/link-relations.xhtml#link-relations-1